### PR TITLE
Sdv tweaks

### PR DESCRIPTION
**Pull request by Stingray540**
Multiple changes to SDV and covvies as requested by XO

### CODE REVIEW

File: code/modules/halo/species/kig-yar.dm [hunk 1]
```diff
@@ -19,10 +19,10 @@
 	spawn_flags = SPECIES_CAN_JOIN
 	flags = NO_MINOR_CUT
 	darksight = 6
-	brute_mod = 1.1
+	brute_mod = 1.5
```
Comment (X0-11) [inline code/modules/halo/species/kig-yar.dm:22]: 1.5 is way too high. A single ma5b round would do 45 damage. Would recommend 1.15 or 1.2

File: maps/first_contact/maps/Covenant Corvette/DAV_Vindicative_Infraction.dmm [hunk 2]
```diff
@@ -389,7 +389,7 @@
 "hy" = (/obj/machinery/light/covenant/east,/turf/simulated/floor/covenant{icon_state = "cov_floor"},/area/covenant_corvette/unggoyquarters)
 "hz" = (/obj/machinery/door/airlock/multi_tile/covenant/three/eastwest{covenant_secure = 1; name = "Secondary Engine"},/turf/simulated/floor/covenant{icon_state = "cov_floor"},/area/covenant_corvette/portengine)
 "hA" = (/obj/structure/window/reinforced/treated{basestate = "invis"; dir = 4; icon_state = "invis"},/turf/simulated/floor/covenant{icon_state = "cov_floor"},/area/covenant_corvette/portengine)
-"hB" = (/obj/machinery/door/airlock/multi_tile/covenant/eastwest{name = "Southern Umbilical"},/turf/simulated/floor/covenant{icon_state = "cov_floor"},/area/covenant_corvette/dockingumbilical)
+"hB" = (/obj/machinery/door/airlock/multi_tile/covenant/eastwest{covenant_secure = 1; name = "Northern Umbilical"},/turf/simulated/floor/covenant{icon_state = "cov_floor"},/area/covenant_corvette/dockingumbilical)
```
Comment (X0-11) [inline maps/first_contact/maps/Covenant Corvette/DAV_Vindicative_Infraction.dmm:392]: This should probably not have changed name.

**Review by X0-11 — CHANGES_REQUESTED** [review]

File: code/modules/halo/species/kig-yar.dm (hunk 1 above)
Comment (BDpuffy420) [reply code/modules/halo/species/kig-yar.dm:22]: Nah let them have it, because they do get dunked after this buff  it legit boils down to “git gud”

Comment (X0-11) [reply code/modules/halo/species/kig-yar.dm:22]: this is barely a buff. They're getting 50 more health in exchange for taking 50% more damage. The health value had more of an effect than just lowering the damage they could take, it also fucked with pain thresholds.

Comment (BDpuffy420) [reply code/modules/halo/species/kig-yar.dm:22]: So if this was merged as is what would be the consequences?

Comment (X0-11) [reply code/modules/halo/species/kig-yar.dm:22]: Kig-yar and t'voan would have technically more health and a higher pain resist cap, but they'd take 50% more damage. A single magnum round would do 90 damage to them.

Let's test it out first. We can tweak it later if it's a big issue.

**Review by X0-11 — APPROVED** [review]

**Pull request merged**